x-pack/test/common/services/security_solution/roles_users_utils.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 import { assertUnreachable } from '../../../../plugins/security_solution/common/utility_types';
-import { FtrProviderContext } from '../../common/ftr_provider_context';
+import { FtrProviderContext } from '../../../detection_engine_api_integration/common/ftr_provider_context';
```

Comment:
This is the module I moved from inside of `detection_engine_api_integrations`. I still need to follow up on this import here and see if I can instead reference the common `ftr_provider_context` instead